From 0abcad150eb1deac67458170514e071a96076bf9 Mon Sep 17 00:00:00 2001
From: Binh Nguyen <binh.nguyen.uw@renesas.com>
Date: Fri, 19 Jul 2019 13:29:05 +0700
Subject: [PATCH 098/458] arm64: dts: r8a774a1-hihope-rzg2m: reserve memory for
 lossy_decompress

lossy_decompress is supported at fixed address defined in IPL and mmngr.
This commit reserve a memory for them.

Signed-off-by: Binh Nguyen <binh.nguyen.uw@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774a1-hihope-rzg2m.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774a1-hihope-rzg2m.dts b/arch/arm64/boot/dts/renesas/r8a774a1-hihope-rzg2m.dts
index 72f9554..c9524fa 100644
--- a/arch/arm64/boot/dts/renesas/r8a774a1-hihope-rzg2m.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774a1-hihope-rzg2m.dts
@@ -29,6 +29,12 @@
 		#size-cells = <2>;
 		ranges;
 
+		/* device specific region for Lossy Decompression */
+		lossy_decompress: linux,lossy_decompress@54000000 {
+			no-map;
+			reg = <0x00000000 0x54000000 0x0 0x03000000>;
+		};
+
 		/* global autoconfigured region for contiguous allocations */
 		linux,cma@58000000 {
 			compatible = "shared-dma-pool";
-- 
2.7.4

